go-controller/pkg/ovn/pods.go

```diff
@@ -86,6 +86,11 @@ func (oc *Controller) deleteLogicalPort(pod *kapi.Pod) {
 		klog.Errorf(err.Error())
 	}
 
+	//FIXME DUAL-STACK
+	if err := oc.lsCache.ReleaseIP(portInfo.logicalSwitch, &portInfo.ip); err != nil {
```

Comment:
ReleaseIP should really be ReleaseIPs and take an array. We'll never release one out of many; it's all or nothing when the pod goes away.

Reply:
That is only germane for dual-stack yes? There is a FIXME added to when we handle dual stack. Similar update needs to be made for Allocate as well when we do dual stack changes.

Reply:
Yeah, only for dual-stack.